src/ServiceControl.Audit/Infrastructure/WebApi/Cors.cs

```diff
@@ -40,13 +40,21 @@ static CorsPolicy GetDefaultPolicy()
                 AllowAnyOrigin = true
             };
 
-            policy.ExposedHeaders.AddRange(new[] { "ETag", "Last-Modified", "Link", "Total-Count", "X-Particular-Version" });
-            policy.Headers.AddRange(new[] { "Origin", "X-Requested-With", "Content-Type", "Accept" });
-            policy.Methods.AddRange(new[] { "POST", "GET", "PUT", "DELETE", "OPTIONS", "PATCH" });
+            AddRange(policy.ExposedHeaders, "ETag", "Last-Modified", "Link", "Total-Count", "X-Particular-Version");
+            AddRange(policy.Headers, "Origin", "X-Requested-With", "Content-Type", "Accept");
+            AddRange(policy.Methods, "POST", "GET", "PUT", "DELETE", "OPTIONS", "PATCH");
 
             return policy;
         }
 
+        static void AddRange(IList<string> list, params string[] items)
+        {
+            foreach (var item in items)
+            {
+                list.Add(item);
+            }
+        }
```

Comment:
Is this the extension method you mentioned? Can it be extracted as an extension method? I suspect there'll be other use-cases for it (maybe)? If not, that's probably fine.

Reply:
I didn't want to cause conflict with the existing extension method in places where we _do_ need to import the RavenDB namespace.